docs/notebooks/expected_improvement.pct.py

```diff
@@ -81,18 +79,13 @@ def build_model(data):
 #
 # The `optimize` method returns several things (see the `optimize` documentation for details), but we're only interested in the data, which captures the points where the objective was queried and the resulting values. Note that the optimizer updates the model in place.
 #
-# However, since the optimization loop catches errors so as not to lose progress, we must check if any errors occurred so we know the data is valid. We'll do that crudely here by re-raising any such errors. You may wish instead to use the history to restore the process from an earlier point.
+# However, the optimization loop catches errors so as not to lose progress, which means the optimization loop might not complete and the data from the last step may not exist. Here we'll handle this crudely by attempting to force the data, which will raise any errors that did occur. For a review of how to handle errors systematically, there is a [dedicated tutorial](recovering_from_errors.ipynb).
```

Comment:
What does "to force the data" mean? not clear to me

Reply:
will clarify